docs/reference/msbuild-targets.md

```diff
@@ -464,6 +465,33 @@ msbuild -t:restore -p:RestorePackagesConfig=true
 > [!NOTE]
 > `packages.config` restore is only available with `MSBuild 16.5+`, and not with `dotnet.exe`
 
+### Restoring with MSBuild static graph evaluation
+
+With MSBuild 16.6+, NuGet has added the capability to use static graph evaluation from the commandline.
+This is an experimental feature that signficantly decreases the msbuild overhead of restore for large repos.
+
+```cli
+msbuild -t:restore -p:RestoreUseStaticGraphEvaluation=true
+```
+
+Alternatively you can enable it by setting the property in a Directory.Build.Props.
```

Comment:
Add an example of perhaps?:

<RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>

Reply:
Sure, I'll add one.